api/tax_calculator/tax_brackets.py

```diff
@@ -1,26 +1,24 @@
 import os
-import json
-from flask import jsonify
 
+from api import utils
 
-brackets_dir = os.path.join(os.path.dirname(__file__), 'fixtures')
+SUPPORTED_TAX_BRACKET_YEARS = [2019, 2020]
```

Comment:
I don't know if this adds any benefit to how the code was already structured.

Reply:
True, but shouldn't `.../tax-calculator/brackets/1111` return a `404: Not Found` instead of a `500: Internal Server Error`